plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/preferences/messages.properties

```diff
@@ -6,5 +6,5 @@ CloudSdkPreferencePage_4=Cannot launch a browser
 CloudSdkPreferencePage_5=&SDK location:
 CloudSdkPreferencePage_6=No SDK found: {0}
 CloudSdkPrompter_0=Google Cloud SDK Not Configured
-CloudSdkPrompter_1=The Google Cloud SDK cannot be found\!  Would you like to configure it now?
+CloudSdkPrompter_1=Invalid or no Google Cloud SDK specified. Do you want to configure Google Cloud SDK now?
```

Comment:
I wanted a message that enforces the user must either specify a valid Cloud SDK or the operation will fail, which the first message didn't convey very well IMO.
I agree the first part doesn't sound great, feel free to provide an alternative.